homeassistant/components/shelly/entity.py

```diff
@@ -20,6 +22,25 @@ def temperature_unit(block_info: dict) -> str:
     return TEMP_CELSIUS
 
 
+def shelly_rest_parser(self, path: str):
+    """Parser for REST path from device status."""
+
+    if "/" not in path:
+        _attribute_value = self.wrapper.device.status[path]
+    else:
+        _attribute_value = self.wrapper.device.status[path.split("/")[0]][
```

Comment:
You're splitting it twice.

Reply:
You prefer to have temp variable for the splitting result instead of doing it twice ?

Reply:
```
        splitted_path = path.split("/")
        _attribute_value = self.wrapper.device.status[splitted_path[0]][splitted_path[1]]
```

Does it work ?